templates/auth_gssapi.conf.erb

```diff
@@ -13,3 +13,16 @@
   ErrorDocument 500 '<html><meta http-equiv="refresh" content="0; URL=/users/login"><body>Kerberos authentication did not pass.</body></html>'
 </Location>
 
+<Location /api/users/extlogin>
+  SSLRequireSSL
+  AuthType GSSAPI
+  AuthName "GSSAPI Single Sign On Login"
+  GssapiCredStore keytab:<%= @http_keytab %>
+  GssapiSSLonly On
+  GssapiLocalName On
+  # require valid-user
+  require pam-account <%= scope.lookupvar('foreman::pam_service') %>
+  ErrorDocument 401 '<html><meta http-equiv="refresh" content="0; URL=/users/login"><body>Kerberos authentication did not pass.</body></html>'
+  # The following is needed as a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1020087
```

Comment:
This was reported against RHEL 6, which used Apache 2.2. Today we use Apache 2.4. Is that bug still really a problem? It was also reported against mod_auth_kerb while today we use mod_gssapi. Perhaps there are better ways now.

I realize this is the same as the old extlogin, but given you have a setup where you can test it's a good chance.

Reply:
Surely I'm missing here something, since I've never touched Apache before :/ But why do we even support `500` here? Does it come from one of the modules just because of some bug?..